ChemoTracker/components/Login/Login.js

```diff
@@ -74,18 +77,26 @@ export default class Login extends React.Component {
         <TextInput
           keyboardType="email-address"
           underlineColorAndroid="transparent"
+          onChangeText={(text)=>{this.setState({loginUsername:text})}}
+          onSubmitEditing={(event) => {
+									this.refs.PasswordField.focus();
```

Comment:
another spacing issue 😛

Reply:
i think its github. i dont have this on my editor. i'll fix it here directly